src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -297,26 +308,32 @@ public StashDiffReport getPullRequestDiffReport(String project, String repositor
   /**
    * Reset all comments linked to a pull-request.
    */
-  public void resetComments(String project, String repository, String pullRequestId, StashDiffReport diffReport, StashUser sonarUser, StashClient stashClient) {
+  public void resetComments(String project, String repository, String pullRequestId,
+                            StashDiffReport diffReport, StashUser sonarUser, StashClient stashClient) {
     try {
+      // Let's call this "diffRep_loop"
       for (StashComment comment : diffReport.getComments()) {
```

Comment:
False-positive that will marked as such after the merge (see: http://sonarqube-archive.15.x6.nabble.com/Sonar-4-4-rule-squid-S135-false-positive-Loops-should-not-contain-more-than-a-single-quot-break-quott-td5028139.html)